tests/cli-v1.rs

```diff
@@ -162,10 +166,10 @@ fn remove_default_toolchain_err_handling() {
     setup(&|config| {
         expect_ok(config, &["rustup", "default", "nightly"]);
         expect_ok(config, &["rustup", "toolchain", "remove", "nightly"]);
-        expect_err(
+        expect_stderr_ok(
             config,
-            &["rustc"],
-            for_host!("toolchain 'nightly-{0}' is not installed"),
+            &["rustc", "--version"],
+            "info: installing component",
```

Comment:
With this change, I feel the function name for the test isn't brilliant.  If you can't be bothered to fix it I don't mind, otherwise it'd be nice if it were renamed something like `remove_default_toolchain_auto_reinstalls` or somesuch

Reply:
`remove_default_toolchain_autoinstalls` is what my fingers put put :P. I noticed after doing it that the very next test, for overridden toolchains, had the same naming bug, but ce la vie.